bump_version.py

```diff
@@ -200,3 +201,16 @@
 
     with open(VERSION_file, 'w', encoding="utf-8") as file:
         file.write(filedata)
+
+## Bump version in builder.sh files
+
+for builder_file in builder_files:
+    with open(builder_file, 'r', encoding="utf-8") as file:
+        print('Bumping version in ' + builder_file)
+        filedata=file.read()
+        # Replace source_branch
+        REGEX=r'source_branch=\"(\d+\.\d+\.\d+)\"'
+        filedata=re.sub(REGEX, f'source_branch=\"{version}\"', filedata)
+
+    with open(builder_file, 'w', encoding="utf-8") as file:
+        file.write(filedata)
```

Comment:
Could we create a function for this block? Looks like we have a multiple instance of this code.

Reply:
The requested change is out of the issue scope, but it's always a good practice. It has been addressed in https://github.com/wazuh/wazuh-packages/pull/2799/commits/abe119a6192d302908e72b17ac5277ff68f2939a.